src/TLuaInterpreter.cpp

```diff
@@ -5855,11 +5855,15 @@ int TLuaInterpreter::searchRoom(lua_State* L)
         if (!roomIdsFound.isEmpty()) {
             for (int i : roomIdsFound) {
                 TRoom* pR = host.mpMap->mpRoomDB->getRoom(i);
-                QString name = pR->name;
-                int roomID = pR->getId();
-                lua_pushnumber(L, roomID);
-                lua_pushstring(L, name.toUtf8().constData());
-                lua_settable(L, -3);
+                // This test is to keep Coverity happy as it thinks pR could be
+                // a nullptr in some odd situation {CID 1415023}:
```

Comment:
Not sure comments like this are really of help.

Reply:
Well I don't *think* it could be a `nullptr` but Coverity has noticed that we *normally* check this (`TRoom*` usages) but we weren't in this case - and I am happy to be more defensive if it helps to improve our currently **A** grade code-rating from them - and linking to the CID might help to track this in that system ...

Reply:
It reads a bit disparaging, maybe if we make it friendlier?